src/post.ts

```diff
@@ -21,14 +21,14 @@ const SKIPPED_FILES = [
  * @param {string} manifestPath;
  * @returns {string}
  */
-async function locateTarget(manifestPath) {
+async function locateTarget(manifestPath: string): Promise<string> {
```

Comment:
Do `async` functions need to return `Promise<T>`, or can they just return `T`?

Reply:
All async functions always return promises, just how they work :)